### PR TITLE
fix: ui container responsiveness

### PR DESCRIPTION
**Pull request by harini-1597**
Fixes # [(200)](https://github.com/pooranjoyb/popShop/issues/200#issuecomment-2154321586)

## Type of change

Please give a X on it which is applicable

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactor Code
- [ ] A documentation update
- [ ] Others(mentioned in the issue number)

## How Has This Been Tested?

Run code locally on machine, use developer tools in Chrome under "Inspect" to check responsiveness on different screens.

## Screenshots and Videos:

https://github.com/pooranjoyb/popShop/assets/128628820/9e236f38-f12c-437e-90f5-2a848f77c4d0

## Checklist:
give a X on it which is applicable

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by pooranjoyb — COMMENTED** [review]
Great job, @harini-1597! 🎉 Thank you for submitting your pull request. Your contribution is valuable and we appreciate your efforts to improve our project.<p>We will promptly review your changes and offer feedback. Keep up the excellent work! Kindly remember to check our [contributing guidelines](https://github.com/pooranjoyb/popShop/blob/dev/CONTRIBUTING.md)</p>

**Comment by harini-1597** [comment]
you can merge the pr

File: src/pages/Home/Home.tsx [hunk 1]
```diff
@@ -58,7 +58,7 @@ function Home() {
       <motion.div
         ref={aboutRef}
         id="about"
-        className="max-w-screen-xl  flex flex-col lg:flex-row justify-between items-center pb-12 pt-24 sm:pt-64 lg:pt-12 w-full"
+        className="max-w-screen-xl flex flex-col lg:flex-row justify-between items-center pb-12 pt-24 sm:pt-64 lg:pt-12 w-full"
```
Comment (Mahitej28) [inline src/pages/Home/Home.tsx:61]: Add `overflow-hidden` value because its leading to a horizontal scroll-bar and that is not needed.

**Review by Mahitej28 — CHANGES_REQUESTED** [review]
@harini-1597 look upon the changes mentioned also @pooranjoyb has told you to fix the `<head />`

**Comment by harini-1597** [comment]
Hey; if you use overflow-hidden the element heading gets cutoff, which was the main UI fix that was to be implemented. And I tried, but I'm unable to really fix that and make the x-scrollbar dissappear.

**Comment by pooranjoyb** [comment]
> Hey; if you use overflow-hidden the element heading gets cutoff, which was the main UI fix that was to be implemented. And I tried, but I'm unable to really fix that and make the x-scrollbar dissappear.

okay np. lemme check with this once.

**Pull request merged**